ui/scss/src/temp.scss

```diff
@@ -416,7 +416,21 @@ div#acn-active-content {
   }
 
   .treenode-label {
-    margin: 0 0 5px 0;
+    height: 100%;
+    position: relative;
+    display: block;
+    & .item-menu {
+      display: none;
+      padding: 3px 10px;
+    }
+
+    &:hover .item-menu {
+      display: block;
+      position: absolute;
+      top: 0;
+      right: 0;
+      cursor: pointer;
+    }
```

Comment:
Are these styles for the ellipsis? Perhaps better to name as such, then. E.g. item-menu-trigger